bob/src/link_manager.rs

```diff
@@ -85,4 +123,16 @@ impl LinkManager {
         })
         .await
     }
+
+    pub(crate) fn update_node_connection(&self, node_name: &str) {
+        if let Some(node) = self.nodes.iter().find(|n| n.name() == node_name) {
+            if let Some(queue) = self.node_check_queue.read().expect("rwlock").as_ref() {
+                if !node.connection_is_set() {
+                    if let Err(e) = queue.try_send(node_name.to_string()) {
+                        warn!("error while updating node {} status: {}", node_name, e);
```

Comment:
Check the error. The queue overflow is not really an error, it's just an indicator that there are too many ping received from disconnected nodes. Change the message accordingly